packages/vat-data/src/far-class-utils.js

```diff
@@ -11,7 +11,8 @@ import {
 
 /** @template L,R @typedef {import('@endo/eventual-send').RemotableBrand<L, R>} RemotableBrand */
 /** @template T @typedef {import('@endo/eventual-send').ERef<T>} ERef */
-/** @typedef {import('@agoric/store').InterfaceGuard} InterfaceGuard */
+// FIXME import InterfaceGuard from @agoric/store
+/** @typedef {*} InterfaceGuard */
```

Comment:
why not do the import now?

Reply:
The type isn't available. Documented in 17aea73588cff36dced0e7c5c1759df30fe25021